files/bootstrap.sh

```diff
@@ -236,7 +236,7 @@ get_resource_to_reserve_in_range() {
   echo $resources_to_reserve
 }
 
-# Calculates the amount of memory to reserve for kubeReserved in mebibytes. KubeReserved is a function of pod
+# Calculates the amount of memory to reserve for kubeReserved in megibytes. KubeReserved is a function of pod
```

Comment:
[From what I can tell](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/), `mebibytes` is actually correct here. A `megabyte` (MB) is 10^6 bytes whereas a `mebibyte` (MiB) is 2^20.